CHANGELOG.md

```diff
@@ -7,6 +7,11 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## unreleased
+
+* Use `GovukPersonalisation::Urls` to generate account URLs ([PR #2289](https://github.com/alphagov/govuk_publishing_components/pull/2289))
+* **BREAKING:** Add dependency on govuk_personalisation, which increases the minimum supported Rails version to 6 ([PR #2289](https://github.com/alphagov/govuk_publishing_components/pull/2289))
```

Comment:
We'd generally just add one `CHANGELOG` entry per PR, and usually have it match the title of the PR. So this would usually be recorded as:
> **BREAKING:** Add a dependency on govuk_personalisation, and use it to generate account URLs in the account layout ([PR #2289](https://github.com/alphagov/govuk_publishing_components/pull/2289))

If it's a breaking change we'd usually make a note of what changes should be made (if any) when one migrates to this version of the gem.

Reply:
I've added a note, though in practice for GOV.UK apps this won't be a breaking change (as we're already on Rails 6)